swarm_copy/tools/get_me_model_tool.py

```diff
@@ -0,0 +1,234 @@
+"""Module defining the Get ME Model tool."""
+
+import logging
+from typing import Any, ClassVar
+
+from langchain_core.tools import ToolException
+from pydantic import BaseModel, Field
+
+from neuroagent.cell_types import get_celltypes_descendants
```

Comment:
We do not want anything coming from `src/`, some utils files have to be added to the `swarm_copy` folder.